catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -441,6 +366,17 @@ private static void updateScriptsToTreeStructure(Project project) {
 		}
 	}
 
+	private static boolean checkValidLanguageVersion(Float languageVersion) {
```

Comment:
i personally dislike "check..." methods because its never really clear what they do without looking it up.

when something returns a boolean, i prefer the method to read as a question, eg. "isValidLanguageVersion" not as a command "checkValidLanguageVersion"